system_tests/bigquery.py

```diff
@@ -117,7 +117,6 @@ def test_create_table(self):
         self.to_delete.insert(0, table)
         self.assertTrue(table.exists())
         self.assertEqual(table.name, TABLE_NAME)
-        self.assertTrue(table._dataset is dataset)
```

Comment:
See the description

> these simply checked how factories created owned objects.

`table` is created a few lines up via

```python
table = dataset.table(TABLE_NAME, schema=[full_name, age])
```

so we aren't learning anything we couldn't learn in unit tests.